sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -256,33 +258,80 @@ export class AssessmentControllerService {
 
   /**
    * Populate the final awards in a dynamic way like disbursement schedule(estimated) awards.
-   * @param disbursementReceipts disbursement receipt details.
+   * @param assessment student assessment.
```

Comment:
Not part of this ticket but this method doesn't have params `disbursementSchedule` and `identifier`.

Reply:
I will remove them.